content/post/kubedb-v2023.08.18/index.md

```diff
@@ -0,0 +1,234 @@
+          requests:
+            cpu: 800m
+            memory: "1Gi"
+  terminationPolicy: WipeOut
+  storageType: Durable
+  version: 3.5.1
+```
+
+**Kafka custom configuration:** With this release support for kafka custom configuration is coming to light. User provided configuration in kubernetes secret will be merged with the default configuration prioritizing the user provided one. Configurations can be provided for both kafka Combined mode and kafka Topology mode for dedicated brokers. Here’s how to configure Kafka with custom configuration secret.
+
+- Create a k8s secret with required configuration file (`server.properties` for combined mode and `broker.properties`/`controller.properties` for dedicated mode).
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: custom-config
+  namespace: demo
+stringData:
+  server.properties: |
+    log.dirs=/var/custom-logdir
+    metadata.log.dir=/var/custom-logdir/metadata
+    min.insync.replicas=2
+```
+- Add the secret reference in Kafka spec.
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: Kafka
+metadata:
+  name: kafka-dev
+  namespace: demo
+spec:
+  configSecret:
+    name: custom-config
+  replicas: 3
+  version: 3.5.1
+  storage:
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 5Gi
+    storageClassName: standard
+  storageType: Durable
+  terminationPolicy: DoNotTerminate
+```
+
+**New version:** support for kafka version `3.4.1` which have some major bugs fixes and `3.5.1` which is the latest one at the time of this release have been added.
+
+## Elasticsearch/OpenSearch:
+
+Previously, Pod Disruption Budget (PDB) was created only when Elasticsearch custom resource spec was provided with `.spec.maxUnavailable` (for combined cluster) or `.spec.topology.<node-type>.maxUnavailable` (for dedicated cluster). This release ensures that Elasticsearch/Opensearch clusters can have at most a single pod from that set that can be unavailable after the eviction. PDB with such configuration is created by default unless it’s a standalone cluster.
+
+**Fixes:** `Internal User credential synchronization for Elasticsearch failure when security is disabled` issue got fixed with this release. `Vertical Scaling not scaling the pod resources` and `Horizontal Scaling Failure in ES v8` issue also have been resolved.
+
+## MongoDB:
+
+**New Version:** Support for MongoDB `4.2.24` have been introduced in this release. Apply the following YAML to try out this new version with KubeDB.
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: MongoDB
+metadata:
+  name: restore
+  namespace: demo
+spec:
+  version: "4.2.24"
+  terminationPolicy: WipeOut
+  storage:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 1Gi
+```
+
+**Fixes:** Use --bind_ip to fix version `3.4.*` CrashLoopbackOff issue.
+Fix for MongoDB HorizontalScale down for shared cluster also have been addressed in this release.
+
+## MySQL:
+
+
+#TODO
```

Comment:
?

Reply:
The description for MySQL changes is not added yet.